background.js

```diff
@@ -0,0 +1,29 @@
+class BackGround {
```

Comment:
backgroundは一般にこれで位置単語であり、 back groundではありません
なので以下のようにします

```suggestion
class Background {
```